third_party/zstd-jni/zstd-jni.BUILD

```diff
@@ -1,18 +1,32 @@
+genrule(
+    name = "copy_link_jni_md_header",
+    srcs = select({
+        "@io_bazel//src/conditions:darwin": ["@bazel_tools//tools/jdk:jni_md_header-darwin"],
```

Comment:
`@bazel_tools//src/conditions:*` is better.

Reply:
updated throughout

Reply:
updated throughout